x-pack/solutions/search/packages/kbn-search-api-keys-components/src/components/api_key_form.tsx

```diff
@@ -46,6 +46,7 @@ export const ApiKeyForm: React.FC<ApiKeyFormProps> = ({ hasTitle = true }) => {
         actions={[
           <EuiButtonIcon
             iconType={status === Status.showPreviewKey ? 'eyeClosed' : 'eye'}
+            size="s"
```

Comment:
Changed to match the built-in `copy` button in `FormInfoField`. We could enforce this via type but I think that's overkill.